must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -94,66 +94,79 @@ for ns in $namespaces; do
     ceph_collection(){
         COMMAND_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/must_gather_commands
         COMMAND_JSON_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/must_gather_commands_json_output
+        COMMAND_LOG_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/logs
```

Comment:
I think it might make sense to name this `COMMAND_ERR_OUTPUT_DIR`? The main purpose of this is to collect the output to fd 2, right?

Reply:
yes. I'll update it